src/plugins/newGuildSettings/index.tsx

```diff
@@ -68,12 +72,50 @@ const settings = definePluginSettings({
     }
 });
 
+const guildPopoutPatch: NavContextMenuPatchCallback = (children, { guild }: { guild: Guild, onClose(): void; }) => {
+    if (!guild) return;
+
+    children.push(
+        <Menu.MenuItem
+            label="Apply NewGuildSettings"
```

Comment:
idk if this is too long xd
```suggestion
            label="Apply Defaults (NewGuildSettings)"
```

Reply:
![image](https://github.com/Vendicated/Vencord/assets/97131358/35195778-1bba-4f8d-8008-37f03106b123)
definitely too long lmao